src/Utilities/StringList.f90

```diff
@@ -63,11 +63,13 @@ function GetStringFromList(list, indx) result(string)
     class(*), pointer :: obj
     type(CharacterStringType), pointer :: charcont
     !
-    string = ''
     obj => list%GetItem(indx)
     charcont => CastAsCharacterStringType(obj)
     if (associated(charcont)) then
+      allocate (character(len=charcont%strlen()) :: string)
       string = charcont
```

Comment:
I think this should be:

```
string(:) = charcont
```

but please verify

Reply:
Will do.

Reply:
The suggestion is to replace line 70 right?